docs/docs/reference/crd-reference/appcontext.md

```diff
@@ -0,0 +1,125 @@
+---
+comments: true
+---
+
+# KeptnAppContext
+
+(Need to add description)
+
+It works with the corresponding
+[KeptnApp](app.md) resource
+that has the same name and is located in the same namespace.
+
+`KeptnAppContext` lists the tasks and evaluations
+to be executed pre/post-deployment.
+Tasks referenced by `KeptnAppContext` are defined in a
+[KeptnTaskDefinition](taskdefinition.md)
+resource.
+`KeptnAppContext` identifies the task by the value of the `metadata.name` field
+and does not need to understand what runner is used to define the task.
+Similarly, evaluations referenced are defined in a
+[KeptnEvaluationDefinition](evaluationdefinition.md)
+resource and identified by the value of the `metadata.name` field;
+`KeptnAppContext` does not need to understand
+the data source or query being used for the evaluation.
+
+## Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnAppContext
+metadata:
+  name: <app-name>
+  namespace: <app-namespace>
+spec:
+  preDeploymentTasks:
+  - <list of tasks>
+  postDeploymentTasks:
+  - <list of tasks>
+  preDeploymentEvaluations:
+  - <list of evaluations>
+  postDeploymentEvaluations:
+  - <list of evaluations>
+  ...
+```
+
+## Fields
+
+* **apiVersion** -- API version being used
+   Must be set to `v1beta1`
+* **kind** -- Resource type
+   Must be set to `KeptnAppContext`
+* **metadata**
+  * **name** -- Unique name of this `KeptnAppContext` resource.
+       Names must comply with the
+       [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+       specification
+       and match the `name` given to the associated `KeptnApp` resource..
+  * **namespace** -- Namespace of this application.
+       This must match the `namespace` given to
+       the associated `KeptnAp` resource.
+
+The remaining fields are required only when implementing
+the release lifecycle management feature.
+If used, these fields must be populated manually:
+
+* **spec**
```

Comment:
also here, missing metadata and spanlinks

Reply:
fixed